packages/plugin-kit/src/createSliceMachineHelpers.ts

```diff
@@ -44,12 +46,31 @@ export class SliceMachineHelpers {
 
 	getProject = async (): Promise<SliceMachineProject> => {
 		const configFilePath = this.joinPathFromRoot("slicemachine.config.json");
-		const configContents = await fs.readFile(configFilePath, "utf8");
-		const config = JSON.parse(configContents);
+
+		let rawConfig: unknown | undefined;
+		try {
+			const contents = await fs.readFile(configFilePath, "utf8");
+			rawConfig = JSON.parse(contents);
+		} catch {
+			// noop
+		}
+
+		if (!rawConfig) {
+			// TODO: Write a more friendly and useful message.
```

Comment:
That's got me thinking, should we ask the Benjamin and Sam about error messages?
.... will the user be seeing these?

Reply:
I'll plan a pass with Sam regarding error messages (all tracked with TODO comments), those are the same as the one in `packages/manager` for now, happy to reopen if needed!